ext/intl/locale/locale.stub.php

```diff
@@ -44,31 +44,31 @@ public static function getKeywords(string $locale) {}
      * @return string|false
      * @alias locale_get_display_script
      */
-    public static function getDisplayScript(string $locale, ?string $in_locale = null) {}
+    public static function getDisplayScript(string $locale, ?string $displayedLocale = null) {}
```

Comment:
Probably a bit controversial, so I'm open for suggestions. `$inLocale` doesn't seem very clear for me (I first thought that `$in_locale` it's an abbreviation for `$input_locale`).

Reply:
Seems reasonable, but should possibly be `$displayLocale` (with the "ed")?